### ISSUE TITLE
Add import functionality to iam_group_policy_attachment and iam_user_policy_attachment

### ISSUE THREAD

**Issue by rwolfson** [terraform-providers/terraform-provider-aws#6602]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

It would be useful if resources `aws_iam_group_policy_attachment` and `aws_iam_user_policy_attachment` had imports implemented, similar to what `aws_iam_role_policy_attachment` currently has. A similar parsing mechanic could be used and I would imagine the logic is about the same. Having this implemented would help facilitate bringing Terraform into live environments where these attachment styles are preferred.

### New or Affected Resource(s)

* aws_iam_group_policy_attachment
* aws_iam_user_policy_attachment

### Potential Terraform Configuration

```
resource "aws_iam_group_policy_attachment" "test-attach" {
  group      = "${aws_iam_group.group.name}"
  policy_arn = "${aws_iam_policy.policy.arn}"
}
```

**Comment by rwolfson** [comment]
That looks great! That looks like it touches `aws_iam_user_policy_attachment`, it also possible to address `aws_iam_group_policy_attachment`? I believe it should be nearly the same change. I can also take a shot at it, but I don't believe I have the environment set up to test to make sure the change works and the tests pass.

**Comment by rwolfson** [comment]
Thank you!

**Comment by bflad** [comment]
Support for importing `aws_iam_group_policy_attachment` and `aws_iam_user_policy_attachment` resources has been merged and will release with version 1.51.0 of the AWS provider, likely middle of this week. 👍

**Comment by bflad** [comment]
This has been released in [version 1.51.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1510-december-05-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
